docs/installation/octopus-in-container/docker-compose-windows.md

```diff
@@ -0,0 +1,184 @@
+SA_PASSWORD=N0tS3cr3t!
+
+# Tag for the Octopus Deploy Server image. Use "latest" to pull the latest image or specify a specific tag
+OCTOPUS_VERSION=2019.13.4
+
+DB_CONNECTION_STRING=Server=db,1433;Initial Catalog=Octopus;Persist Security Info=False;User ID=sa;Password=N0tS3cr3t!;MultipleActiveResultSets=False;Connection Timeout=30;
+
+# The default created user username for login to the Octopus Server
+OCTOPUS_ADMIN_USERNAME=admin
+
+# It is highly recommended this value is changed as it's the default user password for login to the Octopus Server
+OCTOPUS_ADMIN_PASSWORD=Passw0rd123
+
+# Email associated with the default created user. If empty will default to octopus@example.local
+ADMIN_EMAIL=
+
+# Use of this Image means you must accept the Octopus Deploy Eula found here: https://octopus.com/company/legal
+OCTOPUS_ACCEPT_EULA=N
+
+# Use of the SQL Server docker image means you accept the Microsoft SQL Server Eula found here: https://hub.docker.com/r/microsoft/mssql-server-windows-express/
+SQL_SERVER_ACCEPT_EULA=N
+```
+
+In this case, we are specifying the `sa` password that is used when starting the sql container and is used for db connectivity from the Octopus Server. We have also provided the Octopus admin credentials and set a host port mapping to port `81` so that we can access the server externally.
+
+Start both containers by running:
+
+```
+docker-compose --project-name Octopus up -d
+```
+
+When both containers are healthy, you can browse directly to `http://localhost:5441` from your host machine.
+
+## Upgrade with Docker Compose
+
+Upgrades with a Docker Compose project are similar to the steps to [upgrade a single Octopus Server container](octopus-server-container-windows.md). You will still need to get the master key from the original Octopus Server container you used when initially setting up the database. When you have the master key, a simple change to the `.env` file to include the master key and update the Octopus version is all that is required:
+
+```
+SA_PASSWORD=N0tS3cr3t!
+OCTOPUS_ACCEPT_EULA=N
+OCTOPUS_SERVER_TAG=2019.13.4
+DB_CONNECTION_STRING=Server=db,1433;Initial Catalog=Octopus;Persist Security Info=False;User ID=sa;Password=N0tS3cr3t!;MultipleActiveResultSets=False;Connection Timeout=30;
+ADMIN_USERNAME=admin
+ADMIN_PASSWORD=Passw0rd123
+MASTER_KEY=U9ZrQR98uLXyz4CXJzUuCA==
+```
+
+Run the same `docker-compose` command as provided above and Docker will detect that the changes only impact the the Octopus container. It will then stop and recreate only the Octopus Server, leaving the SQL Server running as-is. For further information about the additional configuration of the SQL Server container consult the appropriate [Docker Hub repository information](https://hub.docker.com/r/microsoft/mssql-server-windows-express/) pages. It is generally advised, however, to not run SQL Server inside a container for production purposes.
```

Comment:
```suggestion
Run the same `docker-compose` command as provided above and Docker will detect that the changes only impact the Octopus container. It will then stop and recreate only the Octopus Server, leaving the SQL Server running as-is. For further information about the additional configuration of the SQL Server container consult the appropriate [Docker Hub repository information](https://hub.docker.com/r/microsoft/mssql-server-windows-express/) pages. It is generally advised, however, to not run SQL Server inside a container for production purposes.
```